reolink_aio/api.py

```diff
@@ -919,17 +919,22 @@ async def login(self) -> None:
             self._login_mutex.release()
 
     async def _login_try_ports(self) -> None:
+        first_exc = None
         self._port = 443
         self.enable_https(True)
         try:
             await self.login()
             return
-        except LoginError:
-            pass
+        except LoginError as exc:
+            first_exc = exc
 
         self._port = 80
         self.enable_https(False)
-        await self.login()
+        try:
+            await self.login()
+        except LoginError:
+            # Raise original exception instead of the retry fallback exception
+            raise first_exc
```

Comment:
```suggestion
        except LoginError as exc:
            # Raise original exception instead of the retry fallback exception
            raise first_exc from exc
```